Tests/PackageGraphTests/XCTestManifests.swift

```diff
@@ -66,6 +66,8 @@ extension PubgrubTests {
         ("testResolutionNoConflicts", testResolutionNoConflicts),
         ("testResolutionNonExistentVersion", testResolutionNonExistentVersion),
         ("testResolutionPerformingConflictResolution", testResolutionPerformingConflictResolution),
+        ("testResolutionWithOverridingBranchBasedDependency", testResolutionWithOverridingBranchBasedDependency),
+        ("testResolutionWithSimpleBranchBasedDependency", testResolutionWithSimpleBranchBasedDependency),
```

Comment:
Can you update linux main for the new tests?

Reply:
🤦‍♂️ 
Every. single. time.